go/api/commands.go

```diff
@@ -782,4 +782,21 @@ type ConnectionManagementCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/ping/
 	PingWithMessage(message string) (string, error)
+
+	// Echo the provided message back.
+	// Parameters:
+	//  string message
+	//
+	// Return value:
+	//  The provided message
+	//
+	// For example:
+	//  result, err := client.Echo("Hello World")
+	//	if err != nil {
+	//	    // handle error
+	//	}
+	//	fmt.Println(result.Value()) // Output: Hello World
+	//
+	// [valkey.io]: https://valkey.io/commands/echo/
+	Echo(message string) (Result[string], error)
```

Comment:
note: this should be in another file after merging #2844

Reply:
Waiting for https://github.com/valkey-io/valkey-glide/pull/2844 to be merge.